### PR TITLE
Server: restore voltage/amperage after exit

### LINKED ISSUE #19: Server: restore voltage/amperage after exit to auto
Set auto for Amps, Volts

### PR DESCRIPTION
**Pull request by Lilit1122**
Closes #19

### CODE REVIEW

File: ptd_client_server/server.py [hunk 1]
```diff
@@ -303,21 +351,15 @@ def _handle_cmd(self, cmd: str, p: lib.Proto) -> str:
...
             self._mode = "ranging"
             self._mark = cmd[1]
             return "OK"
         if cmd[0] == "start-testing" and len(cmd) == 2:
             maxVolts, maxAmps = self._ranging_table[cmd[1]]
-            self._ptd.cmd(f"SR,V,{maxVolts}")
-            self._ptd.cmd(f"SR,A,{maxAmps}")
-            with lib.sig:
-                time.sleep(10)
+            self._set_range(f"{maxVolts}", f"{maxAmps}")
```
Comment (xzfc) [inline ptd_client_server/server.py:362]:
```suggestion
            self._set_range(maxVolts, maxAmps)
```

File: ptd_client_server/server.py [hunk 2]
```diff
@@ -289,6 +330,13 @@ def handle_connection(self, p: lib.Proto) -> None:
         finally:
             self._ptd.stop()
 
+    def _set_range(self, volts_value: str, amps_value: str) -> None:
+        self._ptd.cmd(f"SR,V,{volts_value}")
+        self._ptd.cmd(f"SR,A,{amps_value}")
+        logging.info(f"Wait 10 seconds to apply setting for ampere and voltage")
```
Comment (xzfc) [inline ptd_client_server/server.py:336]:
```suggestion
        logging.info("Wait 10 seconds to apply setting for amperage and voltage")
```

File: ptd_client_server/server.py [hunk 3]
```diff
@@ -241,6 +250,38 @@ def cmd(self, cmd: str) -> Optional[str]:
...
+                if (
+                    response_list[param_num] == "0"
+                    and float(response_list[param_num + 1]) > 0
+                ):
+                    return response_list[param_num + 1]
+            except ValueError:
+                logging.info(
+                    "Can not get ptd range value for "
+                    + ("Amps" if param_num == 1 else "Volts")
+                )
```
Comment (xzfc) [inline ptd_client_server/server.py:275]: I think `logging.warning` is more appropriate here. Also, please pass `"Amps"`/`"Volts"` as an argument.

```suggestion
                logging.warning(f"Can not get ptd range value for {setting}")
```

**Review by xzfc — CHANGES_REQUESTED** [review]

File: ptd_client_server/server.py (hunk 2 above)
Comment (Lilit1122) [reply ptd_client_server/server.py:333]: I used "ampere and voltage" term from documentation.(SPEC-PTD-programmers-guide)

**Review by Lilit1122 — COMMENTED** [review]
fixed in e8947ff7e99551bb366cdf6e60790673f33df114

File: ptd_client_server/server.py (hunk 2 above)
Comment (xzfc) [reply ptd_client_server/server.py:333]: OK, but please remove `f` then.

```suggestion
        logging.info("Wait 10 seconds to apply setting for ampere and voltage")
```

**Review by xzfc — APPROVED** [review]

**Pull request merged**